mne/decoding/tests/test_transformer.py

```diff
@@ -229,3 +229,32 @@ def test_unsupervised_spatial_filter():
     usf = UnsupervisedSpatialFilter(PCA(4), average=True)
     usf.fit_transform(X)
     assert_raises(ValueError, UnsupervisedSpatialFilter, PCA(4), 2)
+
+
+def test_temporal_filterer():
+    """Test methods of TemporalFilter."""
+    X = np.random.rand(10, 5, 2)
```

Comment:
2 time samples???

I would see with more than 1000 but take fewer epochs and channels